Requirements/10.sql

```diff
@@ -0,0 +1,78 @@
+--Genre Preference by Age using Group By and Having: 
+--	Determine the preferred genre of different age groups of borrowers. 
+--	(Groups are (0,10), (11,20), (21,30)�)
+
+DROP FUNCTION IF EXISTS fn_BorrowerAge
+
+GO
+
+CREATE FUNCTION fn_BorrowerAge (@BorrowerID INT)
```

Comment:
try to use CREATE OR ALTER instead of dropping and creating functions and sprocs

Reply:
Done